### PR TITLE
Cleaning up syndeo-template/pom.xml

### PR DESCRIPTION
**Pull request by Dippatel98**
Commenting these dependencies for now. I have tested locally and ran a few syndeo pipelines, it ran fine without it. If there are any issue let me know @manavgarg, @nickuncaged1201. If there are no issues, I will remove these in my next commit.

### CODE REVIEW

**Comment by Dippatel98** [comment]
r: @manavgarg , @nickuncaged1201 , @bvolpato

**Review by manavgarg — COMMENTED** [review]
what all pipelines did you run?
can we run some pipelines testing kafka, pubsub, bq and bt maybe to get more confidence?

**Comment by Dippatel98** [comment]
Yeah, sure. I can do that. I ran spanner, kafka, pslite and ps pipelines

File: syndeo-template/pom.xml [hunk 1]
```diff
@@ -234,12 +234,12 @@
             <artifactId>beam-runners-google-cloud-dataflow-java</artifactId>
             <version>${beam.snapshot.version}</version>
         </dependency>
-        <dependency>
+        <!-- <dependency> -->
             <!-- Direct runner included for local development and testing. -->
-            <groupId>org.apache.beam</groupId>
+            <!-- <groupId>org.apache.beam</groupId>
```
Comment (nickuncaged1201) [inline syndeo-template/pom.xml:239]: Not sure if we should remove this? Even though I've never run a pipeline locally, but I don't know if this use case is completely irrelevant

Comment (Dippatel98) [reply syndeo-template/pom.xml:239]: I don't think anyone runs syndeo on direct runner, not even the tests do that.

Comment (Dippatel98) [reply syndeo-template/pom.xml:239]: If there is a valid use case, I can add it back.

**Comment by Dippatel98** [comment]
I ran bq and bt pipelines as well. They seem to work fine w/o the dependencies.

**Comment by manavgarg** [comment]
> I ran bq and bt pipelines as well. They seem to work fine w/o the dependencies.

Thanks. LGTM.

**Review by manavgarg — APPROVED** [review]

**Review by bvolpato — APPROVED** [review]
LGTM.

I would probably remove deps rather than commenting out, we can always go back to this PR and see how it was.

You can do it in 2 phases if it's preferable too, no concerns.

**Pull request merged**